packages/devextreme/scss/widgets/fluent/splitter/_index.scss

```diff
@@ -10,5 +10,12 @@
 .dx-splitter {
   .dx-resize-handle {
     background-color: $splitter-resize-handle-bg;
+    color: $splitter-resize-handle-color;
+  }
+  .dx-state-hover {
+    @include state('hover');
+  }
+  .dx-state-focused {
+    @include state('focus')
```

Comment:
In the mixin, we should pass arguments with color values instead of state

Reply:
Correction: [commit](https://github.com/DevExpress/DevExtreme/pull/26938/commits/a66ce0f2e07b949b174d8f78795b1444e198b393)